__binding__ missing from Qt-stubs

I'm typing checking the following code with mypy

```python
import Qt
Qt.__binding__
```

mypy raises the following error: `Module has no attribute "__binding__"  [attr-defined]`
It looks like `__binding__` is missing from Qt-stubs.

--- a/Qt-stubs/__init__.pyi
+++ b/Qt-stubs/__init__.pyi
@@ -1,4 +1,4 @@
-
+__binding__: str
 __qt_version__: str
 IsPyQt5: bool
 IsPyQt4: bool

